recipe/meta.yaml

```diff
@@ -1,23 +1,23 @@
 {% set name = "backports.csv" %}
 {% set version = "1.0.7" %}
-{% set sha256sum = "1277dfff73130b2e106bf3dd347adb3c5f6c4340882289d88f31240da92cbd6d" %}
+{% set sha256sum = "538e04f193710a7fd2522168e65daa1fd7089766b2721dc5621c9b205c7a285e" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ryanhiebert/backports.csv/archive/master.zip
```

Comment:
Why do you need to GH source? If we can use the PyPI one it usually helps avoid a number of issues, like the checksum change when the releases is modified upstream.

Reply:
@ocefpaf my team has just been told in the past that the github link can be preferred, so it's just my usual choice of formatting. Happy to change if needed

Reply:
At the very least we should avoid using the `master.zip` since that is not released.

Reply:
@ocefpaf @CJ-Wright i made the change back to the PyPi link. thanks

Reply:
> @ocefpaf my team has just been told in the past that the github link can be preferred, so it's just my usual choice of formatting. Happy to change if needed

I'm probably guilty of that one. I do (did?) prefer GH releases b/c they usually had tests and licenses that were not present in the PyPI tarball. However, these cases are becoming more rare and the checksum change on GH releases are quite annoying to deal with.